### PR TITLE
Fixed bug with connecting to console same user

### PR DESCRIPTION
**Pull request by bond95**
Fixes: https://github.com/oVirt/ovirt-web-ui/issues/477

---
This change is [Reviewable](https://reviewable.io/reviews/ovirt/ovirt-web-ui/483)

### CODE REVIEW

File: src/components/VmActions/sagas.js [hunk 1]
```diff
@@ -4,10 +4,11 @@ import { CONSOLE_IN_USE } from './constants'
 import { setConsoleInUse } from './actions'
 
 function* getConsoleInUseModal (sagas, action) {
-  let { vmId, usbFilter } = action.payload
+  let { vmId, usbFilter, userId } = action.payload
   const sessionsInternal = yield sagas.fetchVmSessions({ vmId })
   if (sessionsInternal &&
-    sessionsInternal.find((x) => x.consoleUser) !== undefined) {
+    sessionsInternal.find((x) => x.consoleUser) !== undefined &&
```
Comment (mareklibra) [inline src/components/VmActions/sagas.js]: Can we call `sessionsInternal.find()` just once by extending the predicate?
When refactoring, please keep readability on mind.

**Comment by mareklibra** [comment]
Please add commit message describing the issue and its fix, including link to the bug.

**Comment by bond95** [comment]
@mareklibra Done.

File: src/ovirtapi.js [hunk 2]
```diff
@@ -371,6 +371,9 @@ OvirtApi = {
       return {
         id: c.id,
         consoleUser: c.console_user === 'true',
+        user: {
+          id: c.user.id,
```
Comment (mareklibra) [inline src/ovirtapi.js]: is `c.user !== undefined` always true?

Comment (bond95) [reply src/ovirtapi.js]: Yes, if user is unknown, in that case `user` gonna be:
```
"user": {
   "user_name": "Unknown"
},
```

**Comment by bond95** [comment]
@mareklibra I added `user` check for more safety.

**Pull request merged**